outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "id" {
```

Comment:
Do we need these outputs?

Reply:
If we don't add them, then I cannot use them on examples/outputs.tf.